doc/Data Plane Code Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -1,201 +1,191 @@
 
-````md
-# `Azure.<group>.<service>` Code Generation
+2. Generate client library
 
-Run `dotnet build /t:GenerateCode` to generate code.
+In this section, we'll create a generated API layer built on Azure Core.
 
-### AutoRest Configuration
-> see https://aka.ms/autorest
+From a PowerShell command prompt, navigate to the directory holding `Azure.<group>.<service>.csproj`. Run the following commands:
 
-``` yaml
-input-file:
-- <swagger-file-link>
-namespace: <namespace>
-public-clients: true
-data-plane: true
-security: AADToken
-security-scopes: <AAD-token-scope>
 ```
-````
+sdk\<your-service-name>\Azure.<group>.<service>\src> dotnet build /t:GenerateCode
+```
 
-**Learn more:** see the [README.md](https://github.com/Azure/autorest.csharp/blob/feature/v3/readme.md) to learn more about Azure SDK Code Geneneration.
+After you run the GenerateCode command, you should find a **Generated** folder in your project. Inside the Generated folder you'll find the ServiceClient and ServiceClientOptions which you can use to interact with the service.
 
-#### 2. Run the Generate Command
+3. Export public API
 
-From a PowerShell command prompt, navigate to the directory holding `Azure.<group>.<service>.sln`. Run the following commands:
+If you make public API changes or additions, the `eng\scripts\Export-API.ps1` script has to be run to update public API listings. This generates a file in the library's directory similar to the example found in [sdk\template-dpg\Azure.ServiceTemplate.Template\api\Azure.ServiceTemplate.Template.netstandard2.0.cs](https://github.com/Azure/azure-sdk-for-net/blob/bb0fbccfc33dd27d1ec6f0870022824d47181e61/sdk/template-dpg/Azure.ServiceTemplate.Template/api/Azure.ServiceTemplate.Template.netstandard2.0.cs).
 
+e.g. Running the script for a project in `sdk\deviceupdate` would look like this: 
 ```
-sdk\<your-service-name>\Azure.<group>.<service>\src> dotnet msbuild /t:GenerateCode
+eng\scripts\Export-API.ps1 deviceupdate
 ```
 
-#### 3. Explore the Generated Code
-
-After you've run the `GenerateCode` command above, you should find a `Generated` folder in your `Azure.<group>.<service>` project.
-
-If you're interested, you can look through the swagger file to see how the autorest.csharp extension translated the REST API definition into C# code. This could help you later if you decide to make adjustments to your API design in the swagger file directly, so they can be used for all Tier 1 languages.
-
-**Learn more**: see the [OpenAPI Swagger Documentation](https://swagger.io/docs/) to learn more about swagger file formats.
+## Add package ship requirements
 
-#### 4. Save and commit your changes to your branch
+Before the library package can be released, you will need to add several requirements manually, including tests, samples, README, and CHANGELOG.
+You can refer to following guideline to add those requirements:
+- Tests: https://azure.github.io/azure-sdk/general_implementation.html#testing
```

Comment:
I noticed that there are a number of .md linting errors in this file.  Would you be willing to use the docs-linting and docs-markdown extensions in VSCode to check for valid markdown?

Reply:
resolved